website/docs/cli/commands/graph.mdx

```diff
@@ -53,5 +53,10 @@ by GraphViz:
 $ terraform graph | dot -Tsvg > graph.svg
 ```
 
+Alternatively, `dot` can export to other image file formats by changing the flag input:
```

Comment:
This looks great to me with some small tweaks and an additional line between the sentence and the code block:
```suggestion
Alternatively, `dot` can export other image file formats if you change the flag input:

```